ui/uihandler.go

```diff
@@ -153,6 +153,7 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 	qps, _ := strconv.ParseFloat(r.FormValue("qps"), 64)      // nolint: gas
 	durStr := r.FormValue("t")
 	grpcSecure := (r.FormValue("grpc-secure") == "on")
+	cert := r.FormValue("cert")
```

Comment:
for this I think we should just use the flags (if you want a fortio UI using different cert, start a new one with different cert path)

Reply:
From what I see, the challenge with that approach is:

1. `uihandler.go` creates an instances of the `fgrpc.GRPCRunnerOptions` object.
2. The flags created in `fortio_main.go` are not available to `uihandler.go` or any other pkg for that matter. Therefore, the `Cert` field can not be populated based on any flags.

I believe what you are asking for can be accomplished by putting Fortio flags into a pkg (i.e. cli ) and having `fortio_main.go` and any other dependent pkg's import `cli`. If this has your interest, please create an Issue, assign me and I'll work on refactoring the Fortio cli into a separate pkg. However, I don't think that work should block this PR.

Reply:
we could also pass those global options to ui.handler()

Reply:
Yes, assuming the cli options are in a separate pkg. As I mentioned, I am happy to take on this Fortio cli refactoring. Just create an Issue and assign me. Until then, I think providing a field to enter the cert path is a reasonable option.